libraries/engage/tracking/selectors/cookieConsent.js

```diff
@@ -0,0 +1,58 @@
+import { createSelector } from 'reselect';
+
+/**
+ * Selects the cookie consent modal state.
+ * @param {Object} state The current state of the cookie consent modal.
+ * @returns {Object} whether cookie consent modal is shown.
+ */
+export const getCookieConsentModalState = state => state?.cookieConsent?.cookieConsentModal || {};
+
+/**
+ * Selects the cookie settings state.
+ * @param {Object} state The current state of the cookie settings.
+ * @returns {Object} whether cookies have been selected by the user.
+ */
+export const getCookieSettingsState = state => state?.cookieConsent?.cookieSettings || {};
+
+/**
+ * Selects the property of the comfort cookie settings.
+ * @returns {boolean|null} whether comfort cookies have been selected by the user.
+ */
+export const getAreComfortCookiesActive = createSelector(
```

Comment:
we also need selectors that can be used e.g. by extensions to determine the current settings state. those selectors should return `true` when cookie consent is not active and the state values are `null`.

if you create dedicated selectors for this purpose, you probably should avoid exporting the two existing selectors via `selectors/index.js` to avoid confusion.

Reply:
done